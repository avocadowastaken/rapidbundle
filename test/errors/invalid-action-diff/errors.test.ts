import { testError } from "../../testError";

testError(import.meta.url, { isCI: true });
