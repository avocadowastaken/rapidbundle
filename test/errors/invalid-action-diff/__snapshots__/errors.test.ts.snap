// Vitest Snapshot v1

exports[`invalid-action-diff > output 1`] = `
✔ Resolving build manifests
  ─ ℹ Checking 'action.yml'
  ─ ℹ Checking 'package.json'
  ✔ Run preparations
  ─ ℹ Removing 'dist' directory
  ✘ Making bundle from 'action.yml' [fail]
  ─ ✘ Found build difference:
      --- a/test/errors/invalid-action-diff/dist/index.js
      @@ -1 +0,0 @@
      -console.log("Hello?");
      
      +++ b/test/errors/invalid-action-diff/dist/main.js
      @@ -0,0 +1,4 @@
      +"use strict";
      +
      +// src/main.ts
      +console.log("Main");
      
      +++ b/test/errors/invalid-action-diff/dist/post.js
      @@ -0,0 +1,4 @@
      +"use strict";
      +
      +// src/post.ts
      +console.log("Post");
      
      --- a/test/errors/invalid-action-diff/dist/pre.js
      +++ b/test/errors/invalid-action-diff/dist/pre.js
      @@ -0,0 +1,3 @@
      +"use strict";
      +
      +// src/pre.ts
  ─ ℹ Using '.runs.main' entry: ./src/main.ts
  ─ ℹ Using '.runs.pre' entry: ./src/pre.ts
  ─ ℹ Using '.runs.post' entry: ./src/post.ts
  ─ ℹ Using '.runs.using' entry: node12
  ─ ℹ Checking build difference
------
Exit Code: 1
`;
